clang/test/CIR/Lowering/const.cir

```diff
@@ -0,0 +1,16 @@
+// RUN: cir-tool %s -cir-to-llvm -o %t.mlir
+// RUN: FileCheck --input-file=%t.mlir %s
```

Comment:
Add a case for llvm IR conversion?

Reply:
@htyu I don't see the need to.

There are no CIR attributes being directly lowered to LLVM IR here. They all go through LLVM's MLIR dialect.